[2023-10-14T11:36:09.907+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: bigquery_data_load.create_airflow_dataset manual__2023-10-14T11:36:08.429972+00:00 [queued]>
[2023-10-14T11:36:09.912+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: bigquery_data_load.create_airflow_dataset manual__2023-10-14T11:36:08.429972+00:00 [queued]>
[2023-10-14T11:36:09.912+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2023-10-14T11:36:09.913+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 1
[2023-10-14T11:36:09.913+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2023-10-14T11:36:09.919+0000] {taskinstance.py:1300} INFO - Executing <Task(BigQueryCreateEmptyDatasetOperator): create_airflow_dataset> on 2023-10-14 11:36:08.429972+00:00
[2023-10-14T11:36:09.924+0000] {standard_task_runner.py:55} INFO - Started process 1316 to run task
[2023-10-14T11:36:09.926+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'bigquery_data_load', 'create_***_dataset', 'manual__2023-10-14T11:36:08.429972+00:00', '--job-id', '21', '--raw', '--subdir', 'DAGS_FOLDER/bigquery_data_load.py', '--cfg-path', '/tmp/tmpn35lc81x']
[2023-10-14T11:36:09.927+0000] {standard_task_runner.py:83} INFO - Job 21: Subtask create_***_dataset
[2023-10-14T11:36:09.963+0000] {task_command.py:388} INFO - Running <TaskInstance: bigquery_data_load.create_airflow_dataset manual__2023-10-14T11:36:08.429972+00:00 [running]> on host 4c2fe63b7c62
[2023-10-14T11:36:10.000+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Niko
AIRFLOW_CTX_DAG_ID=bigquery_data_load
AIRFLOW_CTX_TASK_ID=create_***_dataset
AIRFLOW_CTX_EXECUTION_DATE=2023-10-14T11:36:08.429972+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-10-14T11:36:08.429972+00:00
[2023-10-14T11:36:10.001+0000] {base.py:73} INFO - Using connection ID 'google_cloud_default' for task execution.
[2023-10-14T11:36:10.001+0000] {credentials_provider.py:323} INFO - Getting connection using `google.auth.default()` since no key file is defined for hook.
[2023-10-14T11:36:10.003+0000] {bigquery.py:451} INFO - datasetId was not specified in `dataset_reference`. Will use default value vehicle_analytics.
[2023-10-14T11:36:10.003+0000] {bigquery.py:451} INFO - projectId was not specified in `dataset_reference`. Will use default value ***-400718.
[2023-10-14T11:36:10.003+0000] {bigquery.py:461} INFO - Creating dataset: vehicle_analytics in project: ***-400718 
[2023-10-14T11:36:10.797+0000] {bigquery.py:1836} INFO - Dataset vehicle_analytics already exists.
[2023-10-14T11:36:10.826+0000] {taskinstance.py:1323} INFO - Marking task as SUCCESS. dag_id=bigquery_data_load, task_id=create_***_dataset, execution_date=20231014T113608, start_date=20231014T113609, end_date=20231014T113610
[2023-10-14T11:36:10.853+0000] {local_task_job.py:208} INFO - Task exited with return code 0
[2023-10-14T11:36:10.877+0000] {taskinstance.py:2578} INFO - 1 downstream tasks scheduled from follow-on schedule check
